### PR TITLE
feat: help me choose in the header

### PR DESCRIPTION
**Pull request by glee2429**
This PR is to create a CTA to bring users to the personalization quiz lightbox.

### CODE REVIEW

**Review by roger-in-kiva — APPROVED** [review]
Great job!

File: src/pages/Lend/LendHeader.vue [hunk 1]
```diff
@@ -12,6 +12,9 @@
 			<h1 class="tw-mb-2">
 				Make a loan, <br class="hide-for-medium">change a life
 			</h1>
+
+			<a @click="showPersonalizationWizard()" class="tw-cursor-pointer"> Help me choose the perfect loan.</a>
```
Comment (egoroza) [inline src/pages/Lend/LendHeader.vue:16]: could we add a CTA icon here if possible? 😄  check out: 
* [KVMaterialIcon in Storybook](https://608b4cf87f686c00213841b1-ebjvofhmyr.chromatic.com/?path=/docs/kvmaterialicon--default)

and think you can just do like:
```
import KvMaterialIcon from '~/@kiva/kv-components/vue/KvMaterialIcon';
```

and then
```
<kv-material-icon
			:icon="mdiChevronRight"
/>
```

**Review by egoroza — CHANGES_REQUESTED** [review]

**Comment by glee2429** [comment]
As per @egoroza 's recommendation, I added the code block to create an icon following the CTA, but it doesn't show for some reason.  Is there a syntax issue that linter failed to catch? 
Screen Shot 2022-05-19 at 2 29 01 PM

**Comment by roger-in-kiva** [comment]
> As per @egoroza 's recommendation, I added the code block to create an icon following the CTA, but it doesn't show for some reason. Is there a syntax issue that linter failed to catch? Screen Shot 2022-05-19 at 2 29 01 PM

@glee2429 look at this, you need to impor the icon and define it
https://github.com/kiva/ui/blob/df81bed5cc5ed9438d85dd4d97888fd29fbecffc/src/components/LoanCards/KivaClassicBasicLoanCard.vue#L168

https://github.com/kiva/ui/blob/df81bed5cc5ed9438d85dd4d97888fd29fbecffc/src/components/LoanCards/KivaClassicBasicLoanCard.vue#L282

**Comment by glee2429** [comment]
Thank you for the example @roger-in-kiva! The icon appeared but there is a new error message saying that `36:3  error  The "mdiChevronRight" component has been registered but not used`

And I'm very confused because the icon is clearly used as shown below: 

```
			<a @click="showPersonalizationWizard()" class="tw-cursor-pointer"> Help me choose </a>

			<kv-material-icon :icon="mdiChevronRight" />

			<p class="page-subhead show-for-large tw-mb-4">
				Each Kiva loan helps people build a better future for themselves and their families.
			</p>
		</div>
	</div>
</template>

<script>
import ViewToggle from '@/components/LoansByCategory/ViewToggle';
import { mdiChevronRight } from '@mdi/js';
import KvMaterialIcon from '~/@kiva/kv-components/vue/KvMaterialIcon';

export default {
	components: {
		ViewToggle,
		KvMaterialIcon,
		mdiChevronRight
	},
```

**Comment by roger-in-kiva** [comment]
> Thank you for the example @roger-in-kiva! The icon appeared but there is a new error message saying that `36:3 error The "mdiChevronRight" component has been registered but not used`
> 
> And I'm very confused because the icon is clearly used as shown below:
> 
> ```
> 			<a @click="showPersonalizationWizard()" class="tw-cursor-pointer"> Help me choose </a>
> 
> 			<kv-material-icon :icon="mdiChevronRight" />
> 
> 			<p class="page-subhead show-for-large tw-mb-4">
> 				Each Kiva loan helps people build a better future for themselves and their families.
> 			</p>
> 		</div>
> 	</div>
> </template>
> 
> <script>
> import ViewToggle from '@/components/LoansByCategory/ViewToggle';
> import { mdiChevronRight } from '@mdi/js';
> import KvMaterialIcon from '~/@kiva/kv-components/vue/KvMaterialIcon';
> 
> export default {
> 	components: {
> 		ViewToggle,
> 		KvMaterialIcon,
> 		mdiChevronRight
> 	},
> ```
> 
> Screen Shot 2022-05-19 at 2 51 12 PM

Yeah! @glee2429 
The icon should be declared in the data() section as here,
https://github.com/kiva/ui/blob/df81bed5cc5ed9438d85dd4d97888fd29fbecffc/src/components/LoanCards/KivaClassicBasicLoanCard.vue#L282

**Comment by glee2429** [comment]
Got it! I fixed the issue and it passed the test! (?)

**Pull request merged**